int32 message_time
int32 marker_id1
int32 marker_id2
int32 marker_right
int32 marker_left
string objects
string relationship
string selected_qsr
